src/databricks/labs/ucx/install.py

```diff
@@ -170,13 +171,21 @@ def deploy_schema(sql_backend: SqlBackend, inventory_schema: str):
 
 
 class WorkspaceInstaller:
-    def __init__(self, prompts: Prompts, installation: Installation, ws: WorkspaceClient):
+    def __init__(
+        self,
+        prompts: Prompts,
+        installation: Installation,
+        ws: WorkspaceClient,
+        product: str = PRODUCT_INFO.product_name(),
```

Comment:
```suggestion
        product_info: ProductInfo,
        environ: dict[str,str] = os.environ,
```

Reply:
What is the difference between getting it one time from os.environ vs what is suggested here. I am getting flagged when I try to do the above as 
environ = dict(os.environ)
src/databricks/labs/ucx/install.py:174:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)

Reply:
pylint complaining using os.environ as a default value